### ISSUE TITLE
[Naxxramas] Kel'thuzad is missing his enrage.

### ISSUE THREAD

**Issue by Nzzoth** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#5125]
vanilla spell for his enrage: http://www.wow-one.com/database/index.php?spell=28498
wowhead link: http://www.wowhead.com/spell=28498/berserk#used-by-npc

This undeniably proves that KT should have his HP reduced, because there should be an enrage timer for this encounter.

The WotLK version for this encounter is 19 minutes from the start, so the vanilla version should be if not the same, then something like ~20 minutes.

Not having an enrage timer, and this boss having this much HP (almost 9 million) is very far from being Blizzlike. Not to mention that phase 3 is a burn phase. The burn phase having 3million+ HP is also against the design of this encounter, when the original total HP of the boss was 3.15 million (so total HP of boss was less than the feenix's burn phase HP).

HP needs adjusted, enrage needs to be added.
#5122

**Comment by Nzzoth** [comment]
The increased HP is intended.  But yes, an enrage timer should be added.

**Comment by Upset** [comment]
It's probably difficult to design HP values around 16 debuff slots (non-blizzlike.) 
Having only 3 of 8 warlocks using corruption, no COA, etc etc.

**Comment by Rulerx** [comment]
wotlk enrage times are not a good indication; look at the patchwerk enrage notes in http://www.wowwiki.com/Changes_between_Patch_1.11_Naxxramas_and_Patch_3.0_Naxxramas

changed from 6min(vanilla) to 7min(wotlk) but considered much harder in wotlk

**Comment by Cabro** [comment]
Well, there is no video on the internet that shows a KT fight that long from vanilla, that the boss enrages. Or not that i have found. But i've heard of 23 minute fights on Feenix with KT.

**Comment by Nzzoth** [comment]
There were even longer fights then 23 minutes, however can you provide some evidence of what exactly does he do when he enrages? More to it do the adds enrage aswell when he does?

Kind regards, Nzzoth.

**Comment by Cabro** [comment]
Boss's enrage is enough to wipe the raid in 5 seconds. :D

**Comment by ender-feenix** [comment]
@Upset What makes you think 16 debuff slots is not blizzlike?  I'm pretty sure it is...

As for how long the enrage timer should be, what do you all think about 25 minutes?  I think we can safely assume that if you are still in a position to kill him at that point then you're doing something other than what you're supposed to be; even with the increased HP.

**Comment by Upset** [comment]
@ender-feenix 

Here's a video of <Death and Taxes> doing their famous 5-man Loatheb kill during vanilla. You can count as many as 23 debuffs up at various points.

https://www.youtube.com/watch?v=raaLqFelbCk

**Comment by Cabro** [comment]
@ender-feenix if we are to see what they did to the patchwerk enrage timer (added 1 minute), then we should follow the same rule of thumb and make it shorter, or leave it at 19 minutes. all the movies i saw on youtube took about 10 minutes to kill it.

**Comment by Rulerx** [comment]
@ender-feenix 25-30 minutes sounds ok, might need adjustments later on if it's impossible to do it solely due to enrage

@Cabro read my comment above

**Comment by Cabro** [comment]
@Rulerx i read your comment above and it is making the fight custom-like. Lower HP instead and keep it as it was designed as.

**Comment by Nzzoth** [comment]
Alright lets apply some math. MC happens every 55 seconds, on an enrage timer of 20 min that gives you 21.8 MC "phases". Lets count that you handle banshees and abominations left from phase 1 on your first MC "phase" and we will round that number to 20 MC "phases" until he enrages. That being said you have to take 5% of his HP per MC "phase". He has 9 million HP that comes up to 450,000 damage every 55 seconds. Lets say you have 12 healers for that fight which means you have 28 dps (tanks are included into this since they after all deal damage aswell). meaning you need to do 8181 RAID dps per MC (devided by 28 comes to 292 dps per dps). I don't see the standards being set up too high here, but as seen and proven before I can be wrong.

Kind regards, Nzzoth.

**Comment by Rulerx** [comment]
1) tanks should be excluded as well, they are in full prot gear
2) what time step did you use?

**Comment by Cabro** [comment]
Keep in mind that ALL dpsers have to stop doing damage until one of the OTs have it back on him. It's too much, lower the HP.

**Comment by ender-feenix** [comment]
@Upset I'm sorry your comment was deleted by someone.  Would you care to open a new issue and provide your evidence?  We will give it a very close look.

**Comment by killerduki** [comment]
Greetings - isn't much better instead Enrage time to keep it like this high HP because Guardians are some sort of "Enrage" because the longer p3 goes on the stronger Guardians become ;) 

cheers.

**Comment by ender-feenix** [comment]
Um.. what?

**Comment by killerduki** [comment]
I mean with so much HP boss that have now - at p3 the 5 adds that come = longer fight the harder they are.  It would be ok if you don't add the enrage time because with so much HP pool that boss have they become stronger to 1 shot the raid if you waste too much time.

**Comment by Rulerx** [comment]
Guardians become stronger with time only if deaths occur

If no deaths occur, then they are not an enrage

**Comment by meledyne** [comment]
I do not think that 500 flat damage from each attack of KT brings alot of imbalance to think of.
It's not 500%, it won't one-shot tank.
So put it and let it be.

**Comment by Cabro** [comment]
KT's enrage will wipe the raid in 5-6s max. :))

**Comment by Nzzoth** [comment]
Kel'thuzad's enrage increases his damage by 500% (overall not physical). First frostbolt volley everything under 12k hp is dead.

**Comment by meledyne** [comment]
ye, i can see it now
Berserk of KT http://www.wow-one.com/database/index.php?spell=28498
Berserk of Sartura http://www.wow-one.com/database/index.php?spell=27680

it only affects his basic attacks
and lets him cast frostbolt volley every 2 seconds.
well, nevermind then :Q
